atom/browser/api/atom_api_session.cc

```diff
@@ -101,6 +102,19 @@ struct Converter<ClearStorageDataOptions> {
   }
 };
 
+template<>
+struct Converter<content::DownloadItem*> {
+  static v8::Local<v8::Value> ToV8(v8::Isolate* isolate,
+                                   content::DownloadItem* val) {
+    return mate::ObjectTemplateBuilder(isolate)
```

Comment:
`ObjectTemplateBuilder` creates a new ObjectTemplate every time and the call is very expensive, we should just use `Dictionary` in this case.